vector-search-index.md

```diff
@@ -0,0 +1,288 @@
+---
+title: 向量搜索索引
+summary: 了解如何在 TiDB 中构建并使用向量搜索索引加速 K 近邻 (KNN) 查询
+---
+
+# 向量搜索索引
+
+K 近邻 (KNN) 搜索是在向量空间中找到距离给定向量最近的 K 个向量的查询。要找到最近的 K 个向量，最直接的方法就是暴力搜索，即计算向量空间中所有点与给定向量之间的距离。这种方法拥有最好的精确度，但在实际的搜索中，往往搜索速度太慢。因此，近邻搜索问题通常采用近似算法来解决。
+
+在 TiDB 中，你可以创建并利用向量搜索索引来对 [向量类型](/vector-search-data-types.md) 的列进行近似近邻 (ANN) 搜索。通过使用向量搜索索引，整个查询可在几毫秒内完成。
+
+TiDB 目前支持以下向量搜索索引算法：
+
+- HNSW
+
+
+## 创建 HNSW 向量搜索索引
+
+[HNSW](https://en.wikipedia.org/wiki/Hierarchical_navigable_small_world) 是最流行的向量搜索索引算法。HNSW 索引的性能良好，且准确率相对较高 (特定情况下可达 98%),
+
+要创建 HNSW 向量搜索索引，你可以在创建表格时，使用以下语法指定为哪一列的[向量数据类型](/vector-search-data-types.md)构建索引：
+
+```sql
+CREATE TABLE foo (
+    id       INT PRIMARY KEY,
+    data     VECTOR(5),
+    data64   VECTOR64(10),
+    VECTOR INDEX data USING HNSW ((VEC_COSINE_DISTANCE(data)))
```

Comment:
@zimulala, is it ok to create an index with index_name `data` if there is a column with column_name `data`?